packages/components/src/components/Trigger/Trigger.js

```diff
@@ -223,6 +223,15 @@ const Trigger = ({ intl, eventListenerNamespace, trigger }) => {
                 // CEL Interceptor
                 interceptorType = 'CEL';
                 interceptorName = interceptor.cel.filter;
+                const overlays = [];
+                interceptor.cel.overlays.forEach(overlay => {
+                  overlays.push(
+                    <>
+                      <p> expression: {overlay.expression} </p>
+                      <p> key: {overlay.key} </p>
+                    </>
+                  );
+                });
```

Comment:
Items in an array must have a `key` prop, you should see a warning related to this in the browser console when in dev mode. To use this with a fragment we need to use the explicit `<React.Fragment>` instead of the shorthand `<>`

Prefer `.map` instead of `.forEach + .push`.

```js
const overlays = interceptor.cel.overlays.map(overlay => (
  <React.Fragment key={overlay}>
    <p>{expression}</p>
    <p>{key}</p>
  </React.Fragment>
));
```

you'll also need to use react-intl for the labels


I wonder if this is the best presentation of this data, it seems more like a list of some sort (a definition list or table). When there are multiple overlays the current presentation isn't particularly clear at expressing the relationship between the key and value. A structured approach like a table (Carbon `StructuredList` or `DataTable`) makes it more obvious that they're related / part of the same thing as each overlay is captured in a single row.

Reply:
Thanks for the feedback, I've updated all of that now :)